src/WpfMath/TexFormulaParser.cs

```diff
@@ -290,6 +295,65 @@ private static SourceSpan ReadElement(SourceSpan value, ref int position)
 
             return value.Segment(position++, 1);
         }
+        
+        /// <summary>
+        /// Returns the string representation of a measurement argument. A return value indicates whether the argument was valid.
+        /// </summary>
+        /// <param name="value"></param>
+        /// <param name="position"></param>
+        /// <param name="measurementValue"></param>
+        /// <returns></returns>
+        public bool TryReadMeasurementArgument(SourceSpan value,ref int position,out string measurementValue)
+        {
+            if (position == value.Length)
+                throw new TexParseException("illegal end!");
+            SkipWhiteSpace(value, ref position);
+
+            if (value[position]==leftGroupChar)
+            {
+                var measurementGroup=ReadElementGroup(value, ref position, leftGroupChar, rightGroupChar).ToString().Trim();
+
+                if (Regex.IsMatch(measurementGroup, MeasurementPattern))
+                {
+                    if (measurementGroup== Regex.Match(measurementGroup, MeasurementPattern).Value)
+                    {
+                        measurementValue = Regex.Match(measurementGroup, MeasurementPattern).Value;
+                        return true;
+                    }
+                    else
+                    {
+                        measurementValue = null;
+                        return false;
+                    }
+                }
+                else
+                {
+                    measurementValue = null;
+                    return false;
+                }
+            }
+            else if (Regex.Match(value.Segment(position).ToString(),MeasurementPattern).Success)
+            {
+                measurementValue = Regex.Match(value.Segment(position).ToString(), MeasurementPattern).Value;
+
+                if (value.Segment(position).ToString().StartsWith(measurementValue))
+                {
+                    position += measurementValue.Length;
+                    return true;
+                }
+                else
+                {
+                    measurementValue = null;
+                    return false;
+                }
+                
+            }
+            else
+            {
+                measurementValue = null;
+                return false;
+            }
+        }
```

Comment:
I'm sorry but I can't see how these changes contribute to anything. This method is never called.